vcbuild.bat

```diff
@@ -480,22 +481,22 @@ call :run-python tools/cpplint.py %cppfilelist% > nul
 goto exit
 
 :add-to-list
-echo %1 | findstr /c:"src\node_root_certs.h"
+echo %1 | findstr /c:"src\node_root_certs.h" > nul 2>&1
 if %errorlevel% equ 0 goto exit
 
-echo %1 | findstr /r /c:"src\\tracing\\trace_event.h"
+echo %1 | findstr /c:"src\tracing\trace_event.h" > nul 2>&1
 if %errorlevel% equ 0 goto exit
 
-echo %1 | findstr /r /c:"src\\tracing\\trace_event_common.h"
+echo %1 | findstr /c:"src\tracing\trace_event_common.h" > nul 2>&1
 if %errorlevel% equ 0 goto exit
 
-echo %1 | findstr /r /c:"test\\addons\\[0-9].*_.*\.h"
+echo %1 | findstr /r /c:"test\\addons\\[0-9].*_.*\.h" > nul 2>&1
 if %errorlevel% equ 0 goto exit
 
-echo %1 | findstr /r /c:"test\\addons\\[0-9].*_.*\.cc"
+echo %1 | findstr /r /c:"test\\addons\\[0-9].*_.*\.cc" > nul 2>&1
```

Comment:
AIUI if one of these files is found then vcbuild bails out, wouldn't it be helpful to see which one it was?

Reply:
Not the whole `vcbuild`, only the `add-to-list` subroutine. I don't really have a strong opinion about the redirects here, stdout is already discarded when calling `add-to-list`, but it can perhaps help if there's an error in `findstr`. @kfarnung the output with `echo on` is the same, is this helpful for you?

Reply:
@gibfahn This section of code is `call`ed elsewhere in the script.  Because of this it's treated as though it's a separate batch execution.  The early exits here are more akin to early returns in other languages.

@joaocgreis The output is actually to stdout from what I can see.  If you test out `findstr` manually it outputs all successful matches and outputs nothing when it doesn't match.  Since `echo` is being used to pipe into `findstr` I don't believe that `echo on` has any effect here.

The reason for the redirects is that I actually found these counter-intuitive.  I always assumed it was listing the files it was scanning, not the ones that it was ignoring.  If there's strong opposition it might be worth trying to update the output so that it indicates that these are the files being *ignored*, not the ones being scanned.